modules/con-vuln-sources.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+// * cloud_service/acscs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="con-vuln-sources_{context}"]
+= Vulnerability data sources
+
+Sources for vulnerabilities depend on the scanner that is used in your system. {product-title-short} contains two scanners: the StackRox Scanner and Scanner V4. The StackRox Scanner is the default scanner and is deprecated beginning with release 4.6. Scanner V4 was introduced in release 4.4 and is the recommended scanner.
+
+[id="stackrox-scanner-vuln-sources"]
+== StackRox Scanner sources
+
+The StackRox Scanner uses the following vulnerability sources:
+
+* https://access.redhat.com/security/data/oval/v2/[Red{nbsp}Hat OVAL] v2
+* link:https://secdb.alpinelinux.org/[Alpine Security Database]
+* Data tracked in link:https://alas.aws.amazon.com/index.html[Amazon Linux Security Center]
+* link:https://security-tracker.debian.org/tracker/data/json[Debian Security Tracker]
+* link:https://git.launchpad.net/ubuntu-cve-tracker/[Ubuntu CVE Tracker]
+* link:https://nvd.nist.gov/[NVD]: This is used for various purposes such as filling in information gaps when vendors do not provide information. For example, Alpine does not provide a description, CVSS score, severity, or published date.
++
+[NOTE]
+====
+This product uses the NVD API but is not endorsed or certified by the NVD.
+====
+* link:https://github.com/stackrox/scanner/blob/master/ext/vulnsrc/manual/manual.go and https://github.com/stackrox/scanner/blob/master/pkg/vulnloader/nvdloader/manual.go: The upstream StackRox project maintains a set of vulnerabilities that might not be discovered due to data formatting from other sources or absence of data.
+* link:https://www.redhat.com/security/data/metrics/repository-to-cpe.json[repository-to-cpe.json]: Maps RPM repositories to their related CPEs, which is required for matching vulnerabilities for RHEL-based images.
+
+[id="scanner-v4-vuln-sources"]
+== Scanner V4 sources
+
+Scanner V4 uses the following vulnerability sources:
+
+https://access.redhat.com/security/data/csaf/v2/vex/[Red{nbsp}Hat VEX]:: Used with release 4.6 and later. This source provides vulnerability data in link:https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#45-profile-5-vex[Vulnerability Exploitability eXchange(VEX)] format. {product-title-short} takes advantage of VEX benefits to significantly decrease the time needed for the initial loading of vulnerability data, and the space needed to store vulnerability data.
```

Comment:
`link:` prefix?

Also, this was recently updated to `https://security.access.redhat.com/data/csaf/v2/vex/`